roles/ceph-prometheus/tasks/setup_container.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: include ceph-container-common
+  include_role:
```

Comment:
as discussed on IRC, I think it looks a bit agressive to have to run ceph-container-common just to get docker installed in a non containerized deployment, especially when we know the preferred way to run containers in 4.0 will be podman.